src/printer/index.js

```diff
@@ -761,9 +671,11 @@ function genericPrint(path, options, print) {
       return group(concat(parts));
     }
 
+    /* istanbul ignore next */
     default:
-      /* istanbul ignore next */
-      throw new Error("unknown python type: " + JSON.stringify(n.ast_type));
+      // eslint-disable-next-line no-console
+      console.error("Unknown Python node:", n);
+      return n.source;
```

Comment:
What we've done at the beginning of prettier for js was to print `/*! NodeType */source/*! NodeType */` this way it was easy to grep for things that haven't yet been printed. But it's probably not going to work with python since it doesn't have inline comments :(